### PR TITLE
Remove AWX references

### PR DESCRIPTION
**Pull request by sd109**
I'm in the process of writing up some docs on developing custom Azimuth apps and noticed the docs here are outdated. Given I'm not all that familiar with the new CaaS operator, could you give the README a quick read through to check that it still makes sense that and that everything left in does apply to the new non-AWX spec.

### CODE REVIEW

**Comment by sd109** [comment]
Has this appliance ever been tested with the CaaS operator?

**Review by JohnGarbutt — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -58,39 +58,31 @@ repository:
...
     gitUrl: https://github.com/stackhpc/azimuth-sample-appliance.git
-    gitVersion: main
-    metadataRoot: https://raw.githubusercontent.com/stackhpc/azimuth-sample-appliance/main/ui-meta
-    playbooks: [sample-appliance.yml]
+    # The branch, tag or commit id to use
+    gitVersion: master
+    # The name of the playbook to use
+    playbook: sample-appliance.yml
+    # The URL of the metadata file
+    uiMetaUrl: https://raw.githubusercontent.com/stackhpc/azimuth-sample-appliance/master/ui-meta/sample-appliance.yaml
```
Comment (holliefae) [inline README.md]:
```suggestion
    uiMetaUrl: https://raw.githubusercontent.com/stackhpc/azimuth-sample-appliance/master/ui-meta/sample-appliance.yml
```

File: README.md [hunk 2]
```diff
@@ -58,39 +58,31 @@ repository:
...
-  - name: StackHPC Sample Appliance
+azimuth_caas_cluster_templates_overrides:
+  sample-appliance:
+    # The git URL of the appliance
     gitUrl: https://github.com/stackhpc/azimuth-sample-appliance.git
-    gitVersion: main
-    metadataRoot: https://raw.githubusercontent.com/stackhpc/azimuth-sample-appliance/main/ui-meta
-    playbooks: [sample-appliance.yml]
+    # The branch, tag or commit id to use
+    gitVersion: master
```
Comment (holliefae) [inline README.md]:
```suggestion
    gitVersion: main
```

**Review by holliefae — CHANGES_REQUESTED** [review]
Some fixes to README.md

**Pull request merged**